Makefile

```diff
@@ -4,11 +4,12 @@ BINDIR ?= $(DESTDIR)/usr/bin
 BUILDTAGS=
 RUNTIME ?= runc
 COMMIT=$(shell git rev-parse HEAD 2> /dev/null || true)
+VERSION := ${shell cat ./VERSION}
 
 all: tool runtimetest
 
 tool:
-	go build -tags "$(BUILDTAGS)" -ldflags "-X main.gitCommit=${COMMIT}" -o oci-runtime-tool ./cmd/oci-runtime-tool
+	go build -tags "$(BUILDTAGS)" -ldflags "-X main.gitCommit=${COMMIT} -X main.version=${VERSION}" -o oci-runtime-tool ./cmd/oci-runtime-tool
```

Comment:
With this approach there is no way for a Go consumer to figure out which version of the runtime-tools libraries they were built with.  But maybe all serious Go consumers will vendor runtime-tools, so they'll have access to that version via their vendor-pinning file?